test/indexer/test_health_check.py

```diff
@@ -26,21 +23,12 @@ def lambda_name(cls) -> str:
     @mock_sts
     @mock_sqs
     def test_elasticsearch_down(self):
-        self._create_mock_queues()
-        mock_endpoint = ('nonexisting-index.com', 80)
         endpoint_states = self._make_endpoint_states(self.endpoints)
```

Comment:
Why can't this line be part of the extracted method?

Reply:
Because the service health check uses it to generate the expected API endpoints which are passed to it as the first argument: https://github.com/DataBiosphere/azul/pull/1633/files/3749c6fbfb1534e36a3a02475524cecdf0241917#diff-c217391d5a5a682082cd00492ac664a0R55

Reply:
I think we misunderstand each other. The line is identical in both places and is not used outside of the extracted method. I see no need to exclude it from the extracted method. Maybe I'm overlooking something. I'll give this refactoring a shot, that's the only way to resolve this misunderstanding, I think.

Reply:
> and is not used outside of the extracted method

I was wrong. It's return value is used outside.